frontend/src/components/Project/SiteSelection/SiteSelectionViewer.js

```diff
@@ -130,7 +130,7 @@ const SiteSelectionViewer = () => {
               showErrorMessage.isError === false && (
                 <ImageMap
                   header={
-                    "Looking at a map with the source locations and historical events from Geonet in the 2003-present period."
+                    "Source locations and historical events, 2003-present period.(from GeoNet)"
```

Comment:
Should there be a space between the . and ( ?

Reply:
Yeah, Grammarly would warn me if I put these... :thinking: